### PR TITLE
Apply `typeSugar` rule for `swiftformat`

### PR DESCRIPTION
**Pull request by cgrindel**

### CODE REVIEW

File: tools/generator/src/Generator/DisambiguateTargets.swift [hunk 1]
```diff
@@ -41,10 +41,8 @@ extension Generator {
         }
 
         // And then distinguish them
-        var uniqueValues = Dictionary<
-            ConsolidatedTarget.Key,
-            DisambiguatedTarget
-        >(minimumCapacity: targets.count)
+        var uniqueValues = [ConsolidatedTarget.Key:
+            DisambiguatedTarget](minimumCapacity: targets.count)
```
Comment (brentleyjones) [inline tools/generator/src/Generator/DisambiguateTargets.swift:45]:
```suggestion
        var uniqueValues = [
            ConsolidatedTarget.Key: DisambiguatedTarget
        ](minimumCapacity: targets.count)
```

File: tools/generator/test/Fixtures.swift [hunk 2]
```diff
@@ -329,9 +329,7 @@ enum Fixtures {
     static func disambiguatedTargets(
         _ consolidatedTargets: ConsolidatedTargets
     ) -> DisambiguatedTargets {
-        var disambiguatedTargets = Dictionary<
-            ConsolidatedTarget.Key, DisambiguatedTarget
-        >(
+        var disambiguatedTargets = [ConsolidatedTarget.Key: DisambiguatedTarget](
```
Comment (brentleyjones) [inline tools/generator/test/Fixtures.swift:332]:
```suggestion
        var disambiguatedTargets = [
            ConsolidatedTarget.Key: DisambiguatedTarget
        ](
```

Comment (cgrindel) [reply tools/generator/test/Fixtures.swift]: Same comment as above.

File: tools/generator/src/Generator/DisambiguateTargets.swift (hunk 1 above)
Comment (brentleyjones) [reply tools/generator/src/Generator/DisambiguateTargets.swift]: Ohh, if it accepts a trailing comma there, add that too. I was mainly calling out the way the newline was being mis-used.

Comment (cgrindel) [reply tools/generator/src/Generator/DisambiguateTargets.swift]: I was able to remove the directive and add a comma.

**Review by brentleyjones — APPROVED** [review]

**Pull request merged**